semgrep/tests/e2e/snapshots/test_baseline/test_not_git_directory/error.txt

```diff
@@ -1 +1 @@
-Current directory is not in a git project. Aborting baseline scan.
```

Comment:
Non blocking: the old message was more clear on why this failed vs being an invalid commit